html/admin/banpanel.js

```diff
@@ -8,3 +8,12 @@ function toggle_other_checkboxes(source, copycats_str, our_index_str) {
         document.getElementById(source.id.slice(0, -1) + i).checked = source.checked;
     }
 }
+
+function header_click_all_checkboxes(source) {
+	var checkboxes = document.getElementsByClassName(source.name);
+	for (var i = 0, n = checkboxes.length; i < n; i++) {
```

Comment:
```suggestion
	for(var i = 0, n = checkboxes.length; i < n; i++) {
```

please... i cant take the inconsistency... the very next line has no space...